packages/esm-active-visits-app/src/visits-summary/visit-detail.test.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { render, screen, act } from '@testing-library/react';
+import VisitDetailComponent from './visit-detail.component';
+
+jest.mock('./visit.resource', () => ({
+  useVisit: jest.fn(),
+}));
+
+describe('VisitDetailComponent', () => {
+  const visitUuid = 'some-visit-uuid';
+  const patientUuid = 'some-patient-uuid';
```

Comment:
For these kinds of variables, let's use `uuid`s similar to actual `uuid`s. (I think there's a specific format and the number of characters in openmrs uuids)